main/models.py

```diff
@@ -25,8 +25,8 @@ def __str__(self):
         return self.title
 
     def approved_comments(self):
-        """Approving comments. Use for moderating."""
-        return self.comments.filter(approved_comment=True)
+        """Will show comments that are ok in your opinion."""
+        return self.comments.get_approved()
```

Comment:
Okay, it seems I misunderstood the context and the filtering needs to stay here. Please revert related changes.